src/lib/input/input.html

```diff
@@ -34,6 +35,38 @@
              [(ngModel)]="value"
              (change)="_handleChange($event)">
 
+      <textarea
+             #input
+             *ngIf="elementType === 'textarea'"
+             class="md-input-element  md-input-element--textarea"
+             [class.md-end]="align == 'end'"
+             [attr.aria-label]="ariaLabel"
+             [attr.aria-labelledby]="ariaLabelledBy"
+             [attr.aria-disabled]="ariaDisabled"
+             [attr.aria-required]="ariaRequired"
+             [attr.aria-invalid]="ariaInvalid"
+             [attr.autocomplete]="autocomplete"
+             [attr.autocorrect]="autocorrect"
+             [attr.autocapitalize]="autocapitalize"
+             [autofocus]="autofocus"
+             [disabled]="disabled"
+             [id]="inputId"
+             [mdAutosize]="autosize"
+             [attr.max]="max"
+             [attr.maxlength]="maxlength"
+             [attr.min]="min"
+             [attr.minlength]="minlength"
+             [readonly]="readonly"
+             [required]="required"
+             [spellcheck]="spellcheck"
+             [attr.tabindex]="tabindex"
+             [attr.name]="name"
+             (focus)="_handleFocus($event)"
+             (blur)="_handleBlur($event)"
+             [(ngModel)]="value"
+             (change)="_handleChange($event)"></textarea>
```

Comment:
You can remove min, max, and autocorrect. They don't exist on `<textarea>`

We should add `rows`, `cols`, and `wrap`. Need to think about how `rows` interacts with autosize. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea